src/languages/es.ts

```diff
@@ -1919,6 +1919,8 @@ export default {
             customersDescription: 'Elige si queres importar clientes/proyectos y donde los clientes/proyectos son mostrados.',
             locationsDescription: 'Elige si quieres importar lugares y donde los lugares son mostrados.',
             taxesDescription: 'Elige si quires importar las tasas de impuestos y  los impuestos por defecto de tu integración de contaduría.',
+            taxesJournalEntrySwitchNote:
+                'Nota: QuickBooks Online no admite un campo para impuestos en las exportaciones de Anotación en el diario. Cambie su preferencia de exportación a Factura de proveedor o Cheque para importar impuestos.',
```

Comment:
I am not sure how to translate Journal Entry since it's a technical term for Quickbooks Online. I based my translation on this: https://getquipu.com/blog/libro-diario/

```suggestion
                'Nota: QuickBooks Online no admite un campo para impuestos al exportar entradas en el libro diario. Cambia tu preferencia de exportación a Factura de Proveedor o Cheque para importar impuestos.',
```